weblate/machinery/views.py

```diff
@@ -344,7 +344,7 @@ def handle_machinery(request, service, unit, search=None):
     return JsonResponse(data=response)
 
 
-@require_POST
+@require_http_methods(["DELETE", "POST"])
 def translate(request, unit_id, service):
     """AJAX handler for translating."""
     unit = get_object_or_404(Unit, pk=int(unit_id))
```

Comment:
I'm uncertain if it's a good idea to mix this endpoint with removal. Anyway the removal lacks implementation here.

Reply:
For the removal part, I asked a couple of question [here](https://github.com/WeblateOrg/weblate/issues/6440#issuecomment-1186991255)